--- hdf5-1.8.3-snap12/hl/fortran/test/Makefile.in.tstlite	2009-09-21 16:41:49.000000000 -0600
+++ hdf5-1.8.3-snap12/hl/fortran/test/Makefile.in	2009-10-01 14:05:01.628777355 -0600
@@ -355,7 +355,7 @@
 @FORTRAN_SHARED_CONDITIONAL_FALSE@AM_LDFLAGS = -static
 
 # Our main target, the test programs
-TEST_PROG = tstlite tstimage tsttable
+TEST_PROG = tstimage tsttable
 LDADD = $(LIBH5F_HL) $(LIBH5F) $(LIBH5_HL)  $(LIBHDF5)
 
 # Source files for the programs
