--- hdf5-1.6.9/c++/config/commence.in.destdir	2009-04-30 20:22:52.000000000 -0600
+++ hdf5-1.6.9/c++/config/commence.in	2009-06-02 10:48:57.014511985 -0600
@@ -51,12 +51,12 @@
 TRACE=:
 
 ## Installation points
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-bindir=@bindir@
-libdir=@libdir@
-includedir=@includedir@
-docdir=@exec_prefix@/doc
+prefix=$(DESTDIR)@prefix@
+exec_prefix=$(DESTDIR)@exec_prefix@
+bindir=$(DESTDIR)@bindir@
+libdir=$(DESTDIR)@libdir@
+includedir=$(DESTDIR)@includedir@
+docdir=$(DESTDIR)@exec_prefix@/doc
 PUB_LIB=$(LIB)
 
 ## Obtain default library; the current version of libtool does not fully
--- hdf5-1.6.9/fortran/config/commence.in.destdir	2009-04-30 20:22:54.000000000 -0600
+++ hdf5-1.6.9/fortran/config/commence.in	2009-06-02 10:48:42.472512510 -0600
@@ -58,12 +58,12 @@
 TRACE=:
 
 ## Installation points
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-bindir=@bindir@
-libdir=@libdir@
-includedir=@includedir@
-docdir=@exec_prefix@/doc
+prefix=$(DESTDIR)@prefix@
+exec_prefix=$(DESTDIR)@exec_prefix@
+bindir=$(DESTDIR)@bindir@
+libdir=$(DESTDIR)@libdir@
+includedir=$(DESTDIR)@includedir@
+docdir=$(DESTDIR)@exec_prefix@/doc
 PUB_LIB=$(LIB)
 
 ## Shared libraries
--- hdf5-1.6.9/config/commence.in.destdir	2009-04-30 20:22:31.000000000 -0600
+++ hdf5-1.6.9/config/commence.in	2009-06-02 10:47:54.591451873 -0600
@@ -54,12 +54,12 @@
 TRACE=:
 
 ## Installation points
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-bindir=@bindir@
-libdir=@libdir@
-includedir=@includedir@
-docdir=@exec_prefix@/doc
+prefix=$(DESTDIR)@prefix@
+exec_prefix=$(DESTDIR)@exec_prefix@
+bindir=$(DESTDIR)@bindir@
+libdir=$(DESTDIR)@libdir@
+includedir=$(DESTDIR)@includedir@
+docdir=$(DESTDIR)@exec_prefix@/doc
 PUB_LIB=$(LIB)
 
 ## Shared libraries
